app/views/kaminari/_prev_page.html.erb

```diff
@@ -0,0 +1,3 @@
+<span class="prev">
+  <%= link_to_unless current_page.first?, t("views.pagination.previous").html_safe, url, rel: 'prev', remote: remote %>
```

Comment:
Line is too long, and also double quotes :)

Reply:
Done in sha `fcad4ab`